bool ultrasonic_brake_enabled
bool emergency_button_enabled
bool bumper_brake_enabled
bool auto_dock_enabled

bool wheel_release_enabled
bool wheel_brake_enabled

bool ir_enabled


float64[] sonic_distance
